.gitattributes

```diff
@@ -1 +1,3 @@
 *.stl filter=lfs diff=lfs merge=lfs -text
+
+core.autocrlf=true
```

Comment:
No EOF newline :'(

Reply:
does this actually work? i didn't mean add the setting verbatim, just add the attrs that correspond to that setting

Reply:
I think this is actually not needed now that you've fixed the file elsewhere. This autocrlf is kinda just a bodge for working with files with unruly line endings, but since we've now fixed the line endings and added forced normalization, I don't think we need this

Reply:
ah okay i'll remove it

Reply:
agree. delete diff in this file and go off of #45

Reply:
![image](https://user-images.githubusercontent.com/5326007/150384807-ec2a17bf-2e5e-4d57-8d80-a40091912a03.png)

D: